draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -248,17 +248,18 @@ A unique token used in the challenge. It should be a random value issued between
 
 1. MUST have at least 128 bits of entropy.
 2. base64url ({{!RFC4648, Section 5}}) encoded, base32 ({{!RFC4648, Section 6}}) encoded, or base16 ({{!RFC4648, Section 8}}) encoded.
-3. MUST NOT start with an underscore to avoid attacks where tokens are maliciously used as a scope label.
 
 See {{RFC4086}} for additional information on randomness requirements.
 
 Base32 encoding or hexadecimal base16 encoding are RECOMMENDED to be specified when the random token would exist in a DNS label such as in a CNAME target.  This is because base64 relies mixed case (and DNS is case-insensitive as clarified in {{RFC4343}}) and because some base64 characters ("/", "+", and "=") may not be permitted by implementations that limit allowed characters to those allowed in hostnames.  If base32 is used, it SHOULD be specified in way that safely omits the trailing padding ("=").  Note that DNS labels are limited to 63 octets which limits how large such a token may be.
 
 This random token is placed in either the RDATA or a domain name, as described in the rest of this section.  Some methods of validation may involve multiple independent random tokens.
 
+When placed in a domain name, the token MUST NOT start with an underscore to avoid attacks where tokens are maliciously used as a scope label.
```

Comment:
This (which was the same text but in bullet 3 in the previous version) is in-conflict with some of our examples.